stories/Highlight.stories.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { storiesOf } from '@kadira/storybook';
+import { Highlight, Hits } from '../packages/react-instantsearch/dom';
+import { withKnobs, text } from '@kadira/storybook-addon-knobs';
+import { WrapWithHits } from './util';
+
+const stories = storiesOf('Highlight', module);
+
+stories.addDecorator(withKnobs);
+
+const Default = ({ hit }) => (
+  <p>
+    <Highlight attributeName="name" hit={hit} />
+  </p>
+);
+
+Default.propTypes = {
+  hit: React.PropTypes.object.isRequired,
+};
+
+const StrongHits = ({ hit }) => (
+  <p>
+    <Highlight
+      attributeName="name"
+      tagName={text('tag name', 'strong')}
+      hit={hit}
+    />
+  </p>
+);
+
+StrongHits.propTypes = {
+  hit: React.PropTypes.object.isRequired,
+};
+
+stories
+  .add('default', () => (
+    <WrapWithHits hasPlayground={false} linkedStoryGroup="Highlight">
```

Comment:
If you put this prop to true, then the link should appear and it should redirect to the playground story.

At first, those props that indice if you want the "play with props" and "view source code" links where made for the `default` story only because it was embed as an iframe on our doc website. 

I think we should still keep those links only for the default one (being the entry point).